.clang-tidy

```diff
@@ -26,6 +26,7 @@ Checks: >
   performance-*,
   -performance-noexcept-move-constructor,
   readability-*,
+  -readability-function-cognitive-complexity,
```

Comment:
we use some number of macros in our code (OUTCOME_TRY, ERROR, etc). I have a feeling that this check works after all macros are pre-processed which makes it not very useful. Have you looked into this?

My data-point is CLion complaining about cognitive complexity of a small method which uses a lot of OUTCOME_TRYs.

Reply:
nevermind - you meant to disable it not enable.

Reply:
Yeah, sorry, "Add" followed by "-" was not my brightest commit message.